src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/GoogleServicesDAO.scala

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.dsde.firecloud.dataaccess
+
+import java.io.InputStream
+
+import akka.actor.ActorRefFactory
+import org.broadinstitute.dsde.firecloud.model.OAuthTokens
+import spray.http.{HttpRequest, HttpResponse}
+import spray.routing.RequestContext
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait GoogleServicesDAO {
```

Comment:
lifted this and related code from https://github.com/broadinstitute/firecloud-orchestration/pull/312 to make sure we kept things consistent